patterns/header.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Title: Header
+ * Slug: twentytwentyfive/header
+ * Categories: header
+ * Block Types: core/template-part/header
+ * Description: Header with site title and navigation
+ */
+?>
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|30","bottom":"var:preset|spacing|30","left":"var:preset|spacing|50","right":"var:preset|spacing|50"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--30);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--30);padding-left:var(--wp--preset--spacing--50)">
+	<!-- wp:site-title {"level":0} /-->
+	<!-- wp:group {"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group">
```

Comment:
We may want to set some spacing here. If not, there's a gap at the right of the navigation, which makes it look like it's not aligned with the dimensions of the main site wrapper.

<img width="1428" alt="Screenshot 2024-08-20 at 12 19 38" src="https://github.com/user-attachments/assets/143a4acc-7f97-477d-a73b-f23d3b0f18d7">

Reply:
This group is only added to support adding a block hook after the navigation block. Yes it should probably have 0 padding set.

Reply:
Removed, thank you for catching this issue.